.github/workflows/ci_app.yml

```diff
@@ -30,7 +30,8 @@ jobs:
         image: redis:3.2-alpine
         ports: ["6379:6379"]
     env:
-      DB_TEST_DATABASE: pinto_app_test
+      DB_TEST_DATABASE: app-test
```

Comment:
Be careful!

Reply:
It's ok, this way we don't have conflicts when merging the clean-app